src/ORM/Tree/Tree.php

```diff
@@ -50,19 +51,34 @@ public function getRootNodes($rootAlias = 't')
      *
      * @return NodeInterface a node
      */
-    public function getTree($path = '', $rootAlias = 't')
-    {
-        $results = $this->getFlatTree($path, $rootAlias);
+    public function getTree(
+        $path = '',
+        $rootAlias = 't',
+        $materializedPathSeparator = null
+    ) {
+        $materializedPathSeparator = $materializedPathSeparator
+            ?? self::getMaterializedPathSeparator()
```

Comment:
* This repo still requires php >= 5.4.
* If the class using this trait has `getMaterializedPathSeparator()`, for what use case is `$materializedPathSeparator` useful?